game-server/data/static_data/decomposable_items/decomposable_items.xml

```diff
@@ -10849,15 +10849,96 @@
 	<decomposable item_id="188051357" /><!-- Decent bundle of Fossilized Golden Saam -->
 	<decomposable item_id="188051358" /><!-- Big bundle of Fossilized Golden Saam -->
 	<decomposable item_id="188051359" /><!-- Large bundle of Fossilized Golden Saam -->
-	<decomposable item_id="188051360" /><!-- Red Cap Kobushka's Bundle -->
-	<decomposable item_id="188051361" /><!-- Red Cap Kobushka's Zesty Cache -->
-	<decomposable item_id="188051362" /><!-- Red Cap Kobushka's Mystery Cache -->
-	<decomposable item_id="188051363" /><!-- Red Cap Kobushka's Green Cache -->
-	<decomposable item_id="188051364" /><!-- Red Cap Kobushka's Red Cache -->
-	<decomposable item_id="188051365" /><!-- Red Cap Kobushka's Yellow Cache -->
-	<decomposable item_id="188051366" /><!-- Red Cap Kobushka's Copious Cache -->
-	<decomposable item_id="188051367" /><!-- Red Cap Kobushka's Capacious Cache -->
-	<decomposable item_id="188051368" /><!-- Red Cap Kobushka's Colossal Cache -->
+	<decomposable item_id="188051360"><!-- Red Cap Kobushka's Bundle -->
+		<items chance="100.0">
```

Comment:
Decomposables where all items have the same chance do not need a chance property.
Also 100 is the default chance, so this can be removed everywhere.